test/transferSpec.js

```diff
@@ -82,6 +82,50 @@ describe('Plugin transfers (optimistic)', function () {
       }, transferA)).catch(done)
     })
 
+    it('should emit a transfer with correct fields with `outgoing_transfer`', function (done) {
+      const id = uuid()
+
+      this.pluginA.once('outgoing_transfer', (transfer) => {
+        console.log(transfer)
+        assert.equal(transfer.id, id)
+        assert.isOk(transfer.amount)
```

Comment:
I'm not sure. On one hand, there are lots of payment networks that charge a fee. On the other, it seems like that should be added on top of the transfer amount instead of taken out of it. I think it's a plugin-specific thing though.